packages/@lightningjs/ui-components/src/components/Tile/Tile.js

```diff
@@ -175,6 +179,30 @@ export default class Tile extends Surface {
     );
   }
 
+  /* ------------------------------ Icon ------------------------------ */
+
+  _updateIcon() {
+    const iconObject = {
+      w: this.style.iconWidth,
+      h: this.style.iconHeight,
+      icon: this.iconSrc,
+      x: this.style.iconX,
+      mountY: this._isInsetMetadata ? 1 : 0,
+      y: this._isInsetMetadata
+        ? this._metadataY - (this._Metadata ? this._Metadata.h : 0)
+        : this.style.iconYStandard,
```

Comment:
If the design is to always display the providerLogo at the bottom of the Tile, I believe `this.style.iconYStandard` can also be replaced with a calculation using the heights and positions of the Tile and ProgressBar. That is part of how the metadata gets its y position when it is inset(`get _metadataY`). It seems like when `metadataLocation` is `standard`, the logo should be positioned at the same place where metadata is positioned when `inset`, so could use that same calculation:
```js
this._progressBarY
    ? this._progressBarY - this.style.paddingYBetweenContent
    : this._h - this.style.paddingY
```

Reply:
Yeah can do, but I would like to confirm with Josh see if this aligns with the required design?
@joshhowenstine

Reply:
Yes, I believe we should have a `paddingYProgress` equivalent for the new icon